test/entt/entity/organizer.cpp

```diff
@@ -63,15 +63,30 @@ TEST(Organizer, EmplaceFreeFunction) {
     ASSERT_FALSE(graph[2u].top_level());
     ASSERT_FALSE(graph[3u].top_level());
 
-    ASSERT_EQ(graph[0u].children().size(), 2u);
```

Comment:
Since `children` still exists, please, do not drop its tests or the coverage tool will complain. 👍

Reply:
Ping @cjhowedev 🙂